static/nodejs_server/mock_data.ts

```diff
@@ -0,0 +1,1760 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* data used for mock functions for nodejs server tests. */
```

Comment:
wonder how do you generate these mock data? Is there an automated way/script to do this?

Reply:
I did this manually by putting the queries into explore and checking the network responses ..